src/box/lua/schema.lua

```diff
@@ -423,6 +432,11 @@ box.begin = function(options)
         box.rollback()
         box.error()
     end
+    if linearizable and
+       builtin.box_txn_set_is_linearizable(linearizable) ~= 0 then
```

Comment:
A small optimization would be to make `box_txn_set_is_linearizable()` take void args. It does not look like a txn can "revert" its linearizability. Hence it only makes sense to do `box_txn_set_is_linearizable(true)`, and that means the argument isn't needed.

Reply:
Good point, done.